addons/web/static/src/views/calendar/hooks.js

```diff
@@ -97,6 +97,29 @@ export function useFullCalendar(refName, params) {
         } catch (e) {
             throw new Error(`Cannot instantiate FullCalendar\n${e.message}`);
         }
+
+        // Add custom time dot
+        const todayCol = ref.el.querySelector(".fc-day-today.fc-timegrid-col");
+        const indicatorLine = ref.el.querySelector(".fc-timegrid-now-indicator-line");
+
+        if (todayCol && indicatorLine) {
+            const indicatorToday = document.createElement("div");
+            indicatorToday.className = "o_calendar_time_indicator_today";
+            todayCol.appendChild(indicatorToday);
```

Comment:
Why not simply making the today indicator a child element of the line?
Its position would be relative to the line and we don't need the `MutationObserver` anymore.